### DISCUSSION TITLE
I tried to detach the project and found it could not be restored

### DISCUSSION THREAD

**Discussion by HappinerWinnie** [dbeaver/dbeaver#17087] [Q&A]
#### System information: 
- Operating system (distribution) and version
- windows 64
- DBeaver version
- 22.1.0
- Additional extensions

#### Describe the problem you're observing:
I tried to detach the project and found that it could not be restored, but only opened in the window menu bar

I want to restore it to the original workspace, but I can't find any way

#### Steps to reproduce, if exist:
![image](https://user-images.githubusercontent.com/31788525/172751733-fc22e83b-d0bc-46db-9212-67a327562ba9.png)

#### Include any warning/errors/backtraces from the logs
Not necessary

**Comment by ShadelessFox** [comment]
Hello @HappinerWinnie,

Just drag it back like so:

https://user-images.githubusercontent.com/35821147/172774455-453ac43a-0231-4ee4-bbfb-7fc3696e31c3.mp4
